package.json

```diff
@@ -124,6 +124,7 @@
     "@storybook/manager-webpack5": "6.5.15",
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 1 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26527&commitId=5ab15a45e307f6e4e62fe71c442077fc88b62701&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1044 screenshots unchanged, 0 screenshots with different dimensions and 1 screenshots with visible difference.


  <details><summary><b>unknown</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Coachmark.Collapsed.default.chromium.png|159| Changed|

</details>


<div id="vrtCommentfluentuiv8"/>